src/bsoncxx/types.hpp

```diff
@@ -668,6 +708,10 @@ BSONCXX_INLINE bool operator==(const b_maxkey&, const b_maxkey&) {
 #include <bsoncxx/enums/type.hpp>
 #undef BSONCXX_ENUM
 
+BSONCXX_INLINE bool operator!=(const b_utf8& lhs, const b_utf8& rhs) {
```

Comment:
also should be covered by the new b_string methods if there is a typedef alias

Reply:
removed.